README.md

```diff
@@ -11,22 +12,24 @@ Our client requires a simple RESTful API which will be used to perform numeric c
 After our discussions with the client, we have discovered that the solution will:
 - initially support conversions from integers to roman numerals
 - contain endpoints allowing to perform the conversions
-- contain endpoints allowing to fetch conversions performed most frequently or most recently
+- contain endpoints allowing to fetch conversions based on the following criteria:
+  - performed most frequently
+  - performed most recently
```

Comment:
@kubatek94 - I'm concerned that this could have been interpreted as either implement an endpoint to fetch conversions performed most frequently, or an endpoint to fetch conversions performed most frequently. Still not convinced it's quite right though. So further improvements welcomed 😊